controllers/productController.js

```diff
@@ -0,0 +1,12 @@
+const { json } = require("express");
+const { catchAsync } = require("../");
+const { productService } = require("../services/productService");
+
+const getProductsByIdController = async (req, res) => {
```

Comment:
함수명 Controller 부분은 지워도 될 것 같습니다.
```suggestion
const getProductsById = async (req, res) => {
```